x-pack/plugins/ingest_manager/server/services/agent_policy.ts

```diff
@@ -532,10 +532,16 @@ class AgentPolicyService {
       } catch (error) {
         throw new Error('Default settings is not setup');
       }
-      if (!settings.kibana_urls) throw new Error('kibana_urls is missing');
+      if (!settings.kibana_urls || !settings.kibana_urls.length)
+        throw new Error('kibana_urls is missing');
+      const hostsWithoutProtocol = settings.kibana_urls.map((url) => {
+        const parsedURL = new URL(url);
+        return `${parsedURL.host}${parsedURL.pathname !== '/' ? parsedURL.pathname : ''}`;
```

Comment:
@neptunian I'm not sure why this uses pathname in addition to host? Isn't host (hostname + port) enough? I believe it would match the examples added in the tests.